app/controllers/katello/api/v2/capsule_content_controller.rb

```diff
@@ -61,6 +61,12 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, '/capsules/autosync'
```

Comment:
discussed on IRC with @chris1984 that we probably don't need a specific API endpoint just for this setting, can just rely on the core foreman settings API to update this if needed.